### ISSUE TITLE
Strapi Doesn't work when plugin in installed

### ISSUE THREAD

**Issue by arpitjacob** [StellateHQ/strapi-plugin-stellate#1]
Throws this error when I try to start Strapi, uninstalling the plugin fixes it

```ebug: ⛔️ Server wasn't able to start properly.
[2022-12-30 11:20:34.469] error: Could not load js config file 
/node_modules/strapi-plugin-stellate/strapi-server.js: Cannot find module './dist/server'
Require stack:```

**Comment by arpitjacob** [comment]
I had to run `npx tsc` in plugin folder `node_modules/strapi-plugin-stellate` to get to work, looks like the `dist` folder was missing.

**Comment by mxstbr** [comment]
Apologies, fixed in `v0.0.5`!
